src/lsst/rsp/service.py

```diff
@@ -16,9 +16,16 @@ def get_datalink_result(result: ObsCoreRecord) -> DatalinkResults:
 
 def get_siav2_service(label: str) -> SIA2Service:
     """Construct an `SIA2Service` client."""
-    if label != "staff":
-        raise ValueError(label + " data not available at your location")
-
```

Comment:
I think the idea here was, there is no service if it's not a `staff` deployment, and I don't see that happening in the new code.

Reply:
I'm using label here to refer to the data release, as my understanding is that we may have multiple SIAv2 endpoints, one for each data release, so:

```
label = "dp02" -> http://../api/sia/dp02/
label = "dp1" -> http://../api/sia/dp1/
label = "dp1" -> http://../api/sia/dp1/
```
And then the user would request as get_siav2_service("dp02"). Does this not align with what the users would expect? And what is `staff` mean in this context?

Reply:
It's a way to tell "kinds" of RSP installations apart.  "telescope" is T&S (BTS, TTS, Summit).  "science" is IDF.  "staff" is USDF, and the point of this was to make sure you only got an siav2 service at USDF.

Reply:
As it stands we do have an SIAv2 service for the IDFs & for USDF, though not for "telescope".
But I also think we may end up with multiple different endpoints, one per data release, which I'd expect the user to be able to specify in the get_siav2_service call.

For this to work do you think we should have two params to it then? I'm still not sure if we even need the "kind" of RSP installation for this.

Reply:
As long as it does something reasonable if you call it in an environment without an SIAv2 service, I don't think we care how many parameters it has.

Reply:
Ok, I've added a separate parameters for the data-release and kept label to identify the RSP "kind".
For now I just block "telescope" since SIAv2 is available on science & staff.

Reply:
Sorry to do this to you, but...

Let's drop the parameter, and instead rely on checking the environment variable `RSP_SITE_TYPE` with a default of the empty string nor `None` (but do not assume the key is present in the env).  (I am not sure what existing calls we have, but those will need changing).  I would recommend explicitly passing `science` and `staff` and blocking everything else, and here's why: SPHEREx is using lsst-rsp in its UI, we hope it is the first of several-to-many, and (I presume) they will not be setting that variable.

Reply:
I think I've made the change you suggested, have a look and let me know. 
In my change, when `RSP_SITE_TYPE` is not set, I error out, was that what you had in mind?